src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10719,6 +10727,10 @@ namespace UnitarySystems {
         } else {
             this->setAverageAirFlow(state, max(CoolPLR, HeatPLR), OnOffAirFlowRatio);
         }
+        if (FirstHVACIteration && this->m_OAMixerReturnNodeNum > -1) {
+            state.dataLoopNodes->Node(this->m_OAMixerReturnNodeNum).MassFlowRate = state.dataLoopNodes->Node(this->AirInNode).MassFlowRate;
+            MixedAir::ManageOutsideAirSystem(state, compName, FirstHVACIteration, AirLoopNum, this->m_OASysNum);
+        }
```

Comment:
This call is in calcUnitarySystemToLoad which gets called many times throughout the load-based code. This helps keep the mixed air temp in sync with the UnitarySystem inlet. Stomping on the OA Mixer return node flow rate is not a great idea but since this happens only during the FirstHVACIteration then is seems it doesn't hurt anything and subsequent iterations will correct the values. Knowing what the actual system return node flow rate and conditions are after zone equipment simulates is well beyond the scope of a parent object but assumes this initial call helps line up a proper mixed air temperature. I could add an additional if (cyclingfan) here since this is not needed in that case.

Reply:
Other equipment between the mixed air node and the UnitarySystem inlet node is not yet accounted for and therefore the new mixed air conditions would not be present at the UnitarySystem inlet.